app/server.go

```diff
@@ -261,6 +261,21 @@ func NewServer(options ...Option) (*Server, error) {
 		s.StartElasticsearch()
 	}
 
+	s.AddConfigListener(func(oldConfig *model.Config, newConfig *model.Config) {
+		if *oldConfig.GuestAccountsSettings.Enable && !*newConfig.GuestAccountsSettings.Enable {
+			if appErr := s.FakeApp().DeactivateGuests(); appErr != nil {
+				mlog.Warn("Unable to deactivate guest accounts", mlog.Err(appErr))
```

Comment:
Seems like more of an error then a warning. I think the admin will probably want to know this didn't work.

Reply:
fair point

Reply:
done